saber/boss_access/boss_access.py

```diff
@@ -141,529 +110,70 @@ def boss_push_cutout(args):
         rmt = BossRemote("intern.cfg")
 
     # data is desired range
-    if args.bucket:
-        s3 = boto3.resource("s3")
-        with tempfile.TemporaryFile() as f:
-            s3.Bucket(args.bucket).download_fileobj(args.input, f)
-            f.seek(0, 0)
-            data = np.load(f)
-    else:
-        data = np.load(args.input)
-
-    numpyType = np.uint8
-    if args.dtype == "uint32":
-        numpyType = np.uint32
-    elif args.dtype == "uint64":
-        numpyType = np.uint64
-
-    if data.dtype != args.dtype:
-        data = data.astype(numpyType)
-    sources = []
-    if args.source:
-        sources.append(args.source)
+    data = np.load(args.input)
 
     COLL_NAME = args.coll
     EXP_NAME = args.exp
     CHAN_NAME = args.chan
 
+    if args.type == 'image':
```

Comment:
I think it would be better if instead we just let user specify dtype explicitly, because that would allow option for uint16 data OR we can just get is straight from the numpy array. So if dtype is specified we use that but if its not specified we just default to input_array.dtype

Reply:
yes! we know that uint8 or uint16 is image and uint64 is annotation, so we don't need the user to specify image type. Are there any other image types besides 'image' or 'annotation'?